Silicon/ElkhartlakePkg/Library/BdatLib/BdatLib.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+#
+#  Copyright (c) 2019 Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = BdatLib
+  FILE_GUID                      = 3c84a13c-ccdb-4f39-adf8-9799535a77ca
+  MODULE_TYPE                    = BASE
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = BdatLib
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64 IPF
+#
+
+[Sources]
+  BdatLib.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  BootloaderCorePkg/BootloaderCorePkg.dec
+  BootloaderCommonPkg/BootloaderCommonPkg.dec
+  Silicon/ElkhartlakePkg/ElkhartlakePkg.dec
+  Platform/ElkhartlakeBoardPkg/ElkhartlakeBoardPkg.dec

```

Comment:
Please do not have a dependency on Platform package.

Reply:
Done removed.